crypto-common/serializable-state-derive/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "serializable-state-derive"
```

Comment:
It would probably make more sense to call this `crypto-common-derive`, and make it an optional dependency so we don't have hard dependencies on a proc macro stack.

However, for review purposes I would suggest providing it as a followup PR.